src/frontends/tensorflow_common/src/op/segment_sum.cpp

```diff
@@ -45,9 +48,17 @@ OutputVector translate_segment_sum_op(const NodeContext& node) {
     auto num_indices = make_shared<v0::Squeeze>(segment_ids_shape, squeeze_axis);
     auto indices = make_shared<v4::Range>(const_zero, num_indices, const_one, indices_type);
 
+    if (complex_type_mark) {
+        data = complex_type_mark->input_value(0);
+        auto emb_segment_sum_complex = make_shared<v3::EmbeddingSegmentsSum>(data, indices, segment_ids, num_segments);
+        set_node_name(node.get_name(), emb_segment_sum_complex);
+        return {emb_segment_sum_complex->output(0)};
```

Comment:
`SegmentSum` outputs complex type for this case. So we should add complex type mark